### ISSUE TITLE
Some strings refers to closed source add-ons

### ISSUE THREAD

**Issue by AntoineTurmel** [nightingale-media-player/nightingale-l10n#12]
In this string, the software ask the user to install a closed-source add-on for MTP/MSC devices :

https://github.com/nightingale-media-player/nightingale-l10n/blob/master/locales/en-US/songbird/songbird.properties#L1401

We should remove those references and maybe redirect the user to the built-in FolderSync feature.

**Comment by rsjtdrjgfuzkfg** [comment]
Agree. Even if we get MSC or MTP device support at some time, I assume having autoplay is a low-priority feature.

**Comment by AntoineTurmel** [comment]
do we remove those strings ? I think we can remove them, and move all removed strings in an unused file to keep track if they decide to put things open source.

**Comment by rsjtdrjgfuzkfg** [comment]
agree for the unused-strings-file, see issue #17
